pyccel/errors/messages.py

```diff
@@ -123,6 +123,10 @@
 INVALID_FILE_EXTENSION = 'Wrong file extension. Expecting `py` of `pyh`, but found'
 INVALID_PYTHON_SYNTAX = 'Python syntax error'
 
+# ARRAY ERRORS
+ASSIGN_ARRAYS_ONE_ANOTHER = 'assigning arrays one another NOT IMPLEMENTED'
+ARRAY_ALREADY_IN_USE = 'attempt to reallocate an array wich is being used by another variable'
```

Comment:
This message is a lot clearer thanks :+1:
Please capitalize the first word of the sentence
Typo: `wich` -> `which`